server/src/modules/user_Instruments/user_Instrument.entity.ts

```diff
@@ -0,0 +1,15 @@
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import { HydratedDocument, Types } from 'mongoose';
+
+export type User_InstrumentDocument = HydratedDocument<User_Instrument>;
+
+@Schema()
+export class User_Instrument {
+  _id: Types.ObjectId;
+  @Prop({ required: true })
```

Comment:
Should we add the name of the instrument here?

Reply:
indeed